### PR TITLE
Code Review 2 Changes, should compile properly this time

### PR DESCRIPTION
**Pull request by holtcl**
**(This is an update of Holtcl code review2. Unsure what happened with previous one, as it did compile fine at first then suddenly didn't. This one complies properly)**

Analysis of Program:
This program is designed to retrieve compiled data from Covid-19 cases from across the globe and present them to the user in an organized fashion. Overall the program functioned well and appeared to be coded appropriately. Since the last code review they added functionality to the other fragments of the code. The code was very well compiled and I honestly had trouble finding things to critique within it.

Was the Project Available in Github on Time:
Yes

Is the program documented well enough for you to understand:
Yes, the project was very well documented throughout

Does the Program Compile:
Yes

Rational Behind Changes
Per Android Studio's suggestions I changed the text size references from dp to sp. I also added another fragment to bring the bar chart to full screen to allow for easier reading.

Technical Suggestions:
Try to ensure the text is easy to read in all parts of the app. I found the font very small within the bar chart, though it was understandable as the chart was fit onto the screen with other data.
Ensure that the proper values are used when designing the layout. Android Studio gave me warnings about "dp" being used when "sp" was recommended in the xml layout files.
Ensure to utilize unit testing. The only test I saw was an example test
Keep up the good work. Your code is very well put together and well documented.

Technical Concepts I Learned:
How to make a specific object clickable
How to create a full screen display
The use of the News View and Item Adapter

### CODE REVIEW

File: app/src/main/java/edu/uc/groupproject/covid19tracker/MainActivity.kt [hunk 1]
```diff
@@ -2,14 +2,15 @@ package edu.uc.groupproject.covid19tracker
...
+import android.view.View
 import androidx.fragment.app.FragmentTransaction
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.groupproject.covid19tracker.ui.main.ByCountryFragment
 import edu.uc.groupproject.covid19tracker.ui.main.MainFragment
 import edu.uc.groupproject.covid19tracker.ui.main.NewsFragment
 
 class MainActivity : AppCompatActivity() {
-
+    lateinit var barChartFragment: BarChartFragment
```
Comment (holtcl) [inline app/src/main/java/edu/uc/groupproject/covid19tracker/MainActivity.kt:13]: added bar chart variable

File: app/src/main/java/edu/uc/groupproject/covid19tracker/MainActivity.kt [hunk 2]
```diff
@@ -56,4 +57,12 @@ class MainActivity : AppCompatActivity() {
...
         }
     }
+    fun barChartFragment(view: View) {
+        barChartFragment = BarChartFragment()
+        supportFragmentManager
+            .beginTransaction()
+            .replace(R.id.container, barChartFragment)
+            .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
+            .commit()
+    }
```
Comment (holtcl) [inline app/src/main/java/edu/uc/groupproject/covid19tracker/MainActivity.kt:67]: added function to pull up bar chart fragment

File: app/src/main/res/layout/fragment_bar_chart.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textStyle="bold"
+            android:textSize="20sp"
+            android:textColor="#FFF"
+            android:gravity="center"/>
+
+    </androidx.appcompat.widget.Toolbar>
+
+    <androidx.cardview.widget.CardView
```
Comment (holtcl) [inline app/src/main/res/layout/fragment_bar_chart.xml:27]: Bar chart fragment layout

**Comment by sukhbirsekhon** [comment]
We won't add a new fragment at this point. I appreciate you for putting so much effort into it. :)

**Pull request closed without merging**